pkg/aws/awscluster/mutate_awscluster.go

```diff
@@ -6,6 +6,7 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/blang/semver"
```

Comment:
Super cool you found that. We used the CoreOS lib so far but this one looks better and maintained.